anthos-onprem-terraform/avmw_user_cluster_metallb/terraform.tfvars.sample

```diff
@@ -0,0 +1,13 @@
+project_id          = "FLEET_HOST_PROJECT_ID"
```

Comment:
Can you align the variable `=` to be in line?

Reply:
Updated. Thanks for checking!